README.md

```diff
@@ -36,7 +35,7 @@ Open your terminal app. Most likely, you will see a dollar sign on Mac and Unix/
 $ node
 ```
 
-And hit enter. If you have Node installed properly, you'll see a pointy brace `>`. 
+And hit enter. If you have Node installed properly, you'll see a pointy brace `>`.
 
 **Note:** I've seen a few beginners to Node trying to execute shell command in the REPL. For example, creating a folder which is `$ mkdir NAME`. Normal shell commands would not and should not work in REPL, because REPL is a Node environment and it uses Node language, not shell. The giveaway sign of REPL is that you have the angle bracket `>`. Remember that when you run into `SyntaxError: Unexpected identifier` errors. All you need to do is open a new terminal window or exit REPL.
```

Comment:
remove first person reference ("I"). You can change it to something like "Sometimes Node beginners will try to execute terminal commands in the REPL..."